lab/src/Microsoft.ServiceModel.Syndication/src/SyndicationFeedFormatter.cs

```diff
@@ -284,34 +283,34 @@ internal static protected void WriteAttributeExtensions(XmlWriter writer, Syndic
         {
```

Comment:
This method is dangerous. It's a sync method returning void and you are calling and Async method without waiting for it to complete. If the caller tries to use XmlWriter again before feed.WriteAttributeExtensionsAsync has finished using it, then you will end up with a corrupted state.